include/pmtiles.h

```diff
@@ -0,0 +1,57 @@
+/*! \file */ 
+#ifndef _PMTILES_H
+#define _PMTILES_H
+
+#include <fstream>
+#include "external/pmtiles.hpp"
+
+struct TileOffset {
+	uint64_t offset : 40;
+	size_t length : 24;
+
+	TileOffset(uint64_t offset, size_t length) : offset(offset),length(length) { }
+	TileOffset();
+};
+
+// Maximum number of tiles in a leaf directory
+#define LEAF_DIRECTORY_SIZE 10000000
+// Combined size of header and root directory (= start of tile data)
+#define HEADER_ROOT 16384
+// Tile ID at which to start using leaf directories (=z6/0/0)
+#define FIRST_LEAF_TILE 1365
+// Maximum size in bytes of tiles considered "tiny" (i.e. potentially repeatable)
+#define TINY_LENGTH 100
```

Comment:
in planetiler our experiments determined the tile size in bytes isn't well correlated to repeatable tiles because of tag data - instead, detect a small # of square features

Reply:
That's interesting. Should be doable by tweaking the various `tile_worker` code a bit but I think I'll leave that for a further enhancement.